site/documentation/content/admin-guide/coap-adapter-config.md

```diff
@@ -39,6 +42,7 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_COAP_KEY_STORE_PATH`<br>`--hono.coap.keyStorePath` | no | - | The absolute path to the Java key store containing the private key and certificate that the protocol adapter should use for authenticating to clients. Either this option or the `HONO_COAP_KEY_PATH` and `HONO_COAP_CERT_PATH` options need to be set in order to enable TLS secured connections with clients. The key store format can be either `JKS` or `PKCS12` indicated by a `.jks` or `.p12` file suffix respectively. |
 | `HONO_COAP_MAX_CONNECTIONS`<br>`--hono.coap.maxConnections` | no | `0` | The maximum number of concurrent DTLS connections that the protocol adapter should accept. If set to `0`, the protocol adapter determines a reasonable value based on the available resources like memory and CPU. |
 | `HONO_COAP_MAX_PAYLOAD_SIZE`<br>`--hono.coap.maxPayloadSize` | no | `2048` | The maximum allowed size of an incoming CoAP request's body in bytes. Requests with a larger body size are rejected with a 4.13 `Request entity too large` response. |
+| `HONO_COAP_MESSAGE_OFFLOADING`<br>`--hono.coap.messageOffloading` | no | true | Enables to clear payload and serilaized messages in order to reduce the heap consumpion. Experimental. |
```

Comment:
serialized messages from the CoAP stack's deduplicator

Reply:
yes, I will add a reference to the deduplicator.